experimental/dds/tree2/src/core/schema-stored/storedSchemaRepository.ts

```diff
@@ -49,14 +52,16 @@ export interface StoredSchemaRepository extends ISubscribable<SchemaEvents>, Tre
  * not hooked up to Fluid (does not create Fluid ops when editing).
  */
 export class InMemoryStoredSchemaRepository implements StoredSchemaRepository {
-	protected readonly data: MutableSchemaData;
+	protected nodeSchemaData: BTree<TreeNodeSchemaIdentifier, TreeNodeStoredSchema>;
+	protected rootFieldSchemaData: TreeFieldStoredSchema;
 	protected readonly events = createEmitter<SchemaEvents>();
 
 	/**
-	 * For now, the schema are just scored in maps.
-	 * There are a couple reasons we might not want this simple solution long term:
-	 * 1. We might want an easy/fast copy.
-	 * 2. We might want a way to reserve a large namespace of schema with the same schema.
+	 * Copies in the provided schema. If `data` is an InMemoryStoredSchemaRepository, it will be cheap-cloned.
+	 * Otherwise, it will be deep-cloned.
+	 *
+	 * We might not want to store schema in maps long term, as we might want a way to reserve a
+	 * large namespace of schema with the same schema.
```

Comment:
Is that what this sentence means? Not sure what a "large namespace of schema" means precisely.
```suggestion
	 * large space of schema IDs within a schema.
```

Reply:
I think so? I'm preserving (and adding to) a previous comment. That's my interpretation.